<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>847</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_sync">
       <attribute name="title">
        <string>Overview</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>1871</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="labelAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Last Picture Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lastPictureName">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Last Quote Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lastQuoteName">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Last Invoice Name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lastInvoiceName">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Next Quote Name</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Next Invoice Name</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="nextQuoteName">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="nextInvoiceName">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>751</width>
          <height>191</height>
         </rect>
        </property>
        <property name="title">
         <string>Neues Projekt</string>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>62</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Ortschaft</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="newProject_street">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>231</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="newProject_city">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>120</y>
           <width>231</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>62</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Strasse</string>
         </property>
        </widget>
        <widget class="QPushButton" name="createProject">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>150</y>
           <width>151</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Projekt erstellen</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Strassennummer</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>101</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Postleitzahl</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="newProject_streetNumber">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>231</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="newProject_zipCode">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>231</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_projects">
       <attribute name="title">
        <string>Projekte</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Freitext Suche</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Status</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="filterStatus">
              <item>
               <property name="text">
                <string>All</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1 - nothing</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3 - lead</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4 - quoted</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5 - building</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6 - in Betrieb</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7.1 - EIV complete</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7.2 - EIV payed</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8 - archived</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9 - rejected</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="filterFreeText"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QTableWidget" name="tableWidgetProjects">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <column>
               <property name="text">
                <string>Projekt</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Status</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ToDo</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Bauherr</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Anfrage</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>offeriert</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Zusage</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Inbetriebnahme</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Preferences"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>&amp;Save As</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_NewProject">
   <property name="text">
    <string>&amp;New Project</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>newProject_street</tabstop>
  <tabstop>newProject_streetNumber</tabstop>
  <tabstop>newProject_zipCode</tabstop>
  <tabstop>newProject_city</tabstop>
  <tabstop>createProject</tabstop>
  <tabstop>tableWidgetProjects</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
